packages/desktop/src/renderer/containers/hooks.ts

```diff
@@ -68,9 +67,30 @@ export const useCyclingFocus = (
           }
           setCurrentFocus(focusValue)
           break
+        case 'ArrowDown':
+          evt.preventDefault()
+          if (currentFocus === null) {
+            focusValue = 0
+          } else if (currentFocus !== listSize - 1) {
+            focusValue = currentFocus + 1
+          }
+          setCurrentFocus(focusValue)
+          break
+        case 'ArrowUp':
+          evt.preventDefault()
```

Comment:
I think you unnecessarily duplicated the code (identical lines appears under 'PageUp' case). Maybe it's worth it to extract it into a separate function and call it in those two places